build/common.props

```diff
@@ -1,28 +1,32 @@
 <Project>
-
   <PropertyGroup>
+    <!-- we don't want to spawn any-long living compiler processes to avoid BenchmarkDotNet issues with "file in use" -->
     <UseSharedCompilation>false</UseSharedCompilation>
-    <!-- The Python script can narrow down the TFMs to what the user has asked for -->
+    
+    <!-- to make sure that the users don't always need to specify -c Release when using dotnet run -->
```

Comment:
If I specify -c Debug (assuming I have set the BDN attribute that allows that) it properly overrides this, yes?

Reply:
No, it does not override it. I have set it to always true to make sure that the users don't need to specify `-c Release` every time they run the benchmarks (just to make it simpler).